utils/utils.jsx

```diff
@@ -1349,16 +1349,21 @@ export function displayEntireNameForUser(user) {
 }
 
 export function imageURLForUser(userIdOrObject) {
-    return Client4.getUsersRoute() + '/' + userIdOrObject.id + '/image?_=' + (userIdOrObject.last_picture_update || 0);
+    let user = userIdOrObject;
+    if (typeof userIdOrObject === 'string') {
```

Comment:
If `userIdOrObject` is a string, and thus the user id, can we just use it directly in the URL?

Reply:
I tried to get the user id from the state so that we could get `last_picture_update` - not sure what purpose that serves but I figured we wanted that query parameter to be included whenever possible

Reply:
Never mind, just realized we need the `last_picture_update` here too. Hmm.

How painful would it be to not rely on the `store.getState()`? We've been making steady progress towards eliminating global use of the state.